src/tests/multihost/ipa/conftest.py

```diff
@@ -381,10 +381,9 @@ def setup_ipa_client(session_multihost, request):
     ipa_server_uuid = ipa_server.get_default_nw_uuid()
     ipa_server_ip = ipa_server.get_interface_ip(ipa_server_uuid)
     sssd_client.update_resolv_conf(ipa_server_ip)
-
     options = "--ip-address=%s --hostname %s "\
               "--server %s --domain %s "\
-              "--realm %s -w %s -p %s -U" % (ipa_client_ip,
+              "--realm %s -w %s -p %s -U --mkhomedir" % (ipa_client_ip,
```

Comment:
There are pep8 errors for indentation, please fix

Reply:
Fixed